test/multi-sign-test.js

```diff
@@ -0,0 +1,376 @@
+let assert    = require('assert');
+let _         = require('lodash');
+let async     = require('async');
+let testutils = require('./testutils');
+let config    = testutils.init_config();
+let accounts  = require('./testconfig').accounts;
+let Amount    = require('ripple-lib').Amount;
+let Transaction    = require('ripple-lib').Transaction;
+
+suite('MultiSign', function() {
+  let $ = {};
+  let opts = {};
+
+  setup(function(done) {
+    testutils.build_setup(opts).call($, done);
+  });
+
+  setup(function(done) {
+    $.remote.local_signing = false;
+    testutils.create_accounts(
+      $.remote,
+      'root',
+      Amount.from_human('1000 XRP'),
+      ['alice', 'bob', 'carol'],
+      done);
+  });
+
+  teardown(function(done) {
+    testutils.build_teardown().call($, done);
+  });
+
+  function getAliceSequence(callback) {
+    return $.remote.account('alice')._transactionManager._nextSequence;
+  }
+
+  test('remote signing', function(done) {
+    async.series([
+    function(callback) {
+      let tx = $.remote.createTransaction('SignerListSet', {
+        account: accounts.alice.account,
+        signers: [
+          {account: accounts.bob.account, weight: 1},
+          {account: accounts.carol.account, weight: 2 }
+        ],
+        signerQuorum: 3
+      });
+
+      testutils.submit_transaction(tx, callback);
+    },
+
+    function(callback) {
+      let tx = $.remote.createTransaction('AccountSet', {account: accounts.alice.account});
+      tx.setFee(100);
+      tx.setSequence(getAliceSequence());
+      tx.setLastLedgerSequenceOffset(5);
+
+      let mTx = Transaction.from_json(tx.getMultiSigningJson());
+
+      [accounts.bob, accounts.carol].forEach(account => {
+        let signer = mTx.multiSign( account.account, account.secret);
+        assert(signer.Account);
+        assert(signer.SigningPubKey);
+        assert(signer.TxnSignature);
+
+        tx.addMultiSigner(signer);
+      });
```

Comment:
Indent async functions one more step so it's more visually obvious where the `async` list ends.